WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -372,7 +376,18 @@ public static void setAztecEditorEnabled(boolean isEnabled) {
     }
 
     public static boolean isAztecEditorEnabled() {
-        return getBoolean(DeletablePrefKey.AZTEC_EDITOR_ENABLED, true);
+        return isAztecEditorAvailable() && getBoolean(DeletablePrefKey.AZTEC_EDITOR_ENABLED, false);
+    }
+
+    public static void setAztecEditorAvailable(boolean aztecEditorAvailable) {
+        setBoolean(UndeletablePrefKey.AZTEC_EDITOR_AVAILABLE, aztecEditorAvailable);
+        if (aztecEditorAvailable) {
+            AnalyticsTracker.track(Stat.EDITOR_AZTEC_ENABLED);
+        }
+    }
+
+    public static boolean isAztecEditorAvailable() {
+        return getBoolean(UndeletablePrefKey.AZTEC_EDITOR_AVAILABLE, BuildConfig.AZTEC_EDITOR_AVAILABLE);
```

Comment:
Sorry, I meant something like:
```
    public static boolean isAztecEditorAvailable() {
        if (BuildConfig.AZTEC_EDITOR_AVAILABLE) {
            return true;
        }
        return getBoolean(UndeletablePrefKey.AZTEC_EDITOR_AVAILABLE, false);
    }
```

But I just noticed we never call `setBoolean(UndeletablePrefKey.AZTEC_EDITOR_AVAILABLE, false)` so that's pretty much the same code